src/EditorFeatures/Core/CommandHandlers/AbstractCompletionCommandHandler.cs

```diff
@@ -119,6 +120,8 @@ VSCommanding.CommandState IChainedCommandHandler<TypeCharCommandArgs>.GetCommand
         void IChainedCommandHandler<TypeCharCommandArgs>.ExecuteCommand(TypeCharCommandArgs args, Action nextHandler, CommandExecutionContext context)
         {
             AssertIsForeground();
+
+            Logger.Log(FunctionId.Completion_ExecuteCommand_TypeChar, a => a.TypedChar.ToString(), args);
```

Comment:
@olegtk: does this exist in the core editor? If not, can it be? :smile:

Reply:
do you mean the Logger? No, but it would be awesome to have. Please file a bug.